src/platforms/javascript/guides/solid/manual-setup.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Manual Setup
```

Comment:
I think we can remove this, not needed for solidjs

Reply:
oops. bad copy/paste.
Sorry about that!!